README.md

```diff
@@ -388,6 +409,55 @@ Only expose foreign keys for the comment model findById method. eg. `GET /api/co
 - Type: `boolean|array`
 - Default: `false`
 
+## Custom remote
+
+### `jsonapi` remote options
+Sometime you need to control if custom remote should be serialized or not.
```

Comment:
Sometimes you need to be able to control when a custom remote method should be handled by the component. By default, `loopback-component-jsonapi` will not handle (serialize or deserialize) custom remote methods. In order to tell the component to handle a custom remote method, you have the following options (In priority order):

1. Set `jsonapi` to `true` when defining a custom remote method.
2. Add the methods name to the component's `exclude` array setting. (see above)
3. Add the methods name to the component's `include` array setting. (see above)
4. Set `handleCustomRemote` to `true` in the component's settings. (see above)